tool-execution/pmd_setup.sh

```diff
@@ -0,0 +1,41 @@
+## This script sets up PMD on an Ubuntu 20.04 or MacOS 11 machine.
+
+###########
+## SETUP ##
+###########
+
+## checks if the environment variable pmd is set
+if [ !command -v pmd &> /dev/null ]; then
+    echo "pmd already setup on this machine."
+    exit 2
+fi
+
+PMD_DIR=../pmd
+
+## create pmd dir if not exists
+mkdir -p ${PMD_DIR}
+
+## navigate to pmd directory
+cd ${PMD_DIR}
+
+## Check OS version
+OS=$(uname -a | grep -o '\w*' | head -1)
+
+if [ "$OS" == "Darwin" ]; then
+    echo "MacOS detected"
+    echo "Downloading PMD for MacOS..."
+    wget https://github.com/pmd/pmd/releases/download/pmd_releases%2F7.0.0-rc3/pmd-dist-7.0.0-rc3-bin.zip
+elif [ "$OS" == "Linux" ]; then
+    echo "Ubuntu detected"
+    echo "Downloading PMD for Ubuntu..."
+    wget https://github.com/pmd/pmd/releases/download/pmd_releases%2F7.0.0-rc3/pmd-dist-7.0.0-rc3-bin.zip
```

Comment:
The urls here and at line 27 are identical. Why do you need separate code for these two cases?